api/notd/block_processor.py

```diff
@@ -99,18 +102,35 @@ async def _get_retrieved_events(self, blockNumber: int) -> Dict[str, List[Retrie
             transactionHashEventMap[retrievedEvent.transactionHash].append(retrievedEvent)
         return transactionHashEventMap
 
+    async def _get_transaction_weth_values(self, blockNumber: int) -> List[Dict[str, Tuple(str,int)]]:
```

Comment:
why does this return a list? it seems like it should return a dictionary from transaction -> list of weth transfers